src/main/java/bio/terra/service/filedata/google/gcs/GcsBufferedWriter.java

```diff
@@ -0,0 +1,16 @@
+package bio.terra.service.filedata.google.gcs;
+
+import com.google.cloud.storage.Storage;
+import java.io.BufferedWriter;
+import java.nio.channels.Channels;
+import java.nio.charset.StandardCharsets;
+
+/** Given a gs path and a storage object, return a buffered reader for the blob */
+public class GcsBufferedWriter extends BufferedWriter {
```

Comment:
My understanding is that `WriteChannel` is internally buffered so there's no benefit to add more buffering. See `com.google.cloud.BaseWriteChannel`

Reply:
Oh, I didn't realize! I copied the idiom from `GcsBufferedReader`, but it might be prudent to rewrite both of these.

Reply:
The `BufferedReader` provides some nice conveniences of reading strings, lines, etc instead of dealing with character arrays.

Reply:
I think we could provide `0` as the second argument to the `BufferedReader` and `BufferedWriter` constructors to get all the convenience methods without the double-buffering.

Reply:
Nevermind, that very much does not work.

Reply:
Reverting to using raw storage read/write functionality.